core/redis_client.py

```
@@ -0,0 +1,443 @@
+                'request_timestamp': request_timestamp,
+                'user_deadline': user_deadline,
+                'expires_at': expires_at,
+                'user_timelimit_seconds': str(user_timelimit_seconds),
+                'redis_ttl_seconds': str(redis_ttl_seconds)
+            })
+            await self.expire(queue_key, redis_ttl_seconds)
+            
+            # Store hash for quick lookup
+            await self.setex(hash_key, redis_ttl_seconds, hash_value)
+            
+            logger.info(f"Onboarding queued for {mobile_number} with TTL {redis_ttl_seconds}s")
```

Ruff (0.14.6)

Logging statement uses f-string

(G004)